helper/flags/flags_test.go

```diff
@@ -91,6 +91,13 @@ func TestMultiAddrFromDns(t *testing.T) {
 			err:        true,
 			outcome:    "",
 		},
+		{
+			name:       "valid long domain suffix",
+			dnsAddress: "dns/validator-1.foo.technology",
+			port:       12345,
```

Comment:
I know it's a nitpick, but could you please extract this `12345` port value into a common value within the test? 🙏 

We use it for multiple test cases, and it should not be a magic number here.

Reply:
https://github.com/0xPolygon/polygon-edge/pull/405/commits/8b028ec5ad1f2a597ee4696a7a9933217eb43b60 Resolved here